releasetasks/templates/funsize.yml.tmpl

```diff
@@ -0,0 +1,175 @@
+{% for partial_version in partial_updates.keys() %}
```

Comment:
Hmm, good point! Looks like partial_version (and maybe its build number) need to be part of the basename for funsize tasks and anything downstream of it.